### DISCUSSION TITLE
V Rising Server

### DISCUSSION THREAD

**Discussion by vincss** [GameServerManagers/LinuxGSM#3865] [General]
Hi !
Did anyone succeed to host a V Rising dedicated server using LinuxGSM ?
Thanks. :-)

**Comment by AndrewLuebke** [comment]
I'm also curious about this one.

**Comment by RedCarl** [comment]
+1

**Comment by jkavenaugh** [comment]
https://docs.linuxgsm.com/faq

_Please also check that the server can run on Linux before submitting. Servers that require WINE will not be considered. You can use SteamDB.info to find the server and check Linux comparability._

:(

I followed some guide online to get mine going under wine.

**Comment by vincss** [comment]
Hi !
I made a small app that allow to start/stop the dedicated server and a tuto to run VRising under Linux : 
https://github.com/vincss/sleeping-linux-gsm/wiki/%5BHowTo%5D-V-Rising

Don't hesitate pour add your comments.
